packages/demo/examples/08-network/ExpandableNetwork.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint class-methods-use-this: 0, react/no-unused-prop-types: 0 */
```

Comment:
do you need either of these?  (if you remove `getStateFromProps` `class-methods-use-this` shouldn't throw errors any more)